execution/executor-types/src/ledger_update_output.rs

```diff
@@ -22,7 +22,7 @@ use std::sync::Arc;
 pub struct LedgerUpdateOutput {
     pub statuses_for_input_txns: Vec<TransactionStatus>,
     pub to_commit: Vec<TransactionToCommit>,
-    pub reconfig_events: Vec<ContractEvent>,
+    pub events: Vec<ContractEvent>,
```

Comment:
can we only have events for notification here?

Reply:
fixed